app/models/concerns/json_web_token.rb

```diff
@@ -0,0 +1,15 @@
+require "jwt"
+module JsonWebToken
+  extend ActiveSupport::Concern
+    SECRET_KEY = Rails.application.secrets.secret_key_base. to_s

```

Comment:
```suggestion
    SECRET_KEY = Rails.application.secrets.secret_key_base.to_s

```

This me - coz I wasn't sure kung mali ba o monitor ko lol 

![image](https://user-images.githubusercontent.com/780659/191939912-2cae8070-e1f5-42be-9f00-b745aaf6970d.png)

Reply:
Hahaha diko din napansin sorry.